### DISCUSSION TITLE
Your request was approved. Your benefits will be available soon.

### DISCUSSION THREAD

**Discussion by Vidhan0206** [community/community#137265] [GitHub Education]
### Hi everyone,

Congrats! Your request for [@js-vidhan]was approved and your benefits will be available [soon]
 Its been almost 5 days but still it is showing this please help!!

**Comment by MiguelCardoso19** [comment]
Wait a bit longer and check your email. If nothing changes, contact GitHub Support(https://support.github.com/request) for help.

**Reply by atharmehar** [reply]
How much time does it take to get the benefits?

**Comment by MiguelCardoso19** [comment]
Benefits usually take a few days to process after approval. Wait more 2 days just in case or contact Support via the link i provided.

**Comment by davevad93** [comment]
Hi @js-vidhan ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @js-vidhan was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
